fitbenchmarking/fitbenchmarking
# Add Accuracy and Runtime to the Fitting Reports

#### Description of Work
This PR adds the accuracy and runtime of a fit to its Fitting report:
![image](https://user-images.githubusercontent.com/40830825/145050487-49c058ff-3850-47ef-9173-a8accac601a7.png)

Fixes #998

#### Testing Instructions

1. Run any fit and make sure the accuracy and runtime is displayed

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
